pure-pso/README.md

```diff
@@ -9,7 +9,7 @@ This helm chart installs the Pure Service Orchestrator CSI plugin on a Kubernete
 4. Currently, there is **no upgrade supported** from previous versions that do not deploy the datastore (PSO 5.x and lower).
 5. You **MUST** supply a unique `clusterID` in values.yaml. This was previously called `namespace.pure`. `clusterID` must be less than or equal to 22 characters in length. `clusterID` must be unique between **all** Kubernetes clusters using your Pure devices or naming conflicts will result. **WARNING** Do not change `clusterID` once it has been set during the initial installation of PSO on a cluster.
 6. `helm uninstall` will perform the initial uninstallation, but some pods will continue to clean up post-installation. They should go away after cleanup is complete.
-7. Note that PSO CSI only supports the Beta version snapshotter APIs. The snapshotter CRDs for the Beta version APIs have been upgraded, therefore use only release-2.0 CRDs as detailed below.
+7. Note that PSO CSI only supports the Beta version snapshotter APIs. The snapshotter CRDs for the Beta version APIs have been upgraded, therefore use only release-2.0 CRDs as detailed below. In addition, the snapshotclass will be automatically applied to the cluster if user has specified 'openshift' as the orchestrator in values.yaml.
```

Comment:
Tiny nit, I wouldn't mind this being `VolumeSnapshotClass` (CamelCase and the full CRD name)